apps/dot_ext/forms.py

```diff
@@ -88,16 +92,15 @@ class Meta:
     def clean(self):
         client_type = self.cleaned_data.get("client_type")
         authorization_grant_type = self.cleaned_data.get("authorization_grant_type")
-
         msg = ""
         validate_error = False
 
-        # Validate choices
+        #Validate choices
         if not (
             client_type == "confidential"
             and authorization_grant_type == "authorization-code"
         ):
-            validate_error = True
+            #validate_error = True
             msg += (
                 "Only a confidential client and "
                 "authorization-code grant type are allowed at this time."
```

Comment:
I think you can cut this down through line 113, since you're hard-coding these values.

Reply:
actually, 95-113 can go, really.

Reply:
I cleaned up and pushed the changes.